koku/masu/external/downloader/azure/azure_report_downloader.py

```diff
@@ -41,6 +41,105 @@ class AzureReportDownloaderNoFileError(Exception):
     """Azure Report Downloader error for missing file."""
 
 
+def get_initial_dataframe_with_delta(local_file, manifest_id, provider_uuid, start_date, context, tracing_id):
+    """
+    Fetch initial dataframe from CSV plus start_delta and time_inteval.
+
+    Args:
+        local_file (str): The full path name of the file
+        manifest_id (str): The manifest ID
+        provider_uuid (str): The uuid of a provider
+        filepath (str): The full path name of the file
+        start_date (Datetime): The start datetime of incoming report
+        context (Dict): Logging context dictionary
+        tracing_id (str): The tracing id
+    """
+    dh = DateHelper()
+    try:
+        data_frame = pd.read_csv(local_file, usecols=["UsageDateTime"])
+        time_interval = "UsageDateTime"
+        date_format = "%Y-%m-%d %H:%M:%S"
+        base_cols = utils.INGRESS_REQUIRED_COLUMNS
+    except ValueError:
```

Comment:
Do we want a debug log here, or some other way to indicated an exception was handled?